src/sample/Main.java

```diff
@@ -29,9 +29,6 @@ public void start(Stage primaryStage)
 
     static void setupView(View gameView)
     {
-        // creation of the grid pane
-        gameView.setupTheGridPane();
-
         // setting up the buttons which go into the UI
         gameView.setupButtons(
```

Comment:
We do need to make note to work getting this pattern completely out of the project. It is getting there.

Reply:
But also the two remaining ones are not even related. one is make move and the other is return to startup gui so I would say if we do not have a task for this then we need to.